src/revivalpmmp/pureentities/PureEntities.php

```diff
@@ -255,13 +255,14 @@ public static function create($type, Position $source, ...$args) {
             return Entity::createEntity($type, $source->getLevel(), $nbt, ...$args);
         } else {
             $chunk = $source->getLevel()->getChunk($source->x >> 4, $source->z >> 4, true);
+			$level = $source->getLevel();
             if (!$chunk->isGenerated()) {
                 $chunk->setGenerated();
             }
             if (!$chunk->isPopulated()) {
                 $chunk->setPopulated();
             }
-            return Entity::createEntity($type, $chunk, $nbt, ...$args);
```

Comment:
The IF statement is now not needed anymore. I saw that Tesseract has been updated. So the IF clause can be removed and only Level should be used!